File: src/main/java/net/softbell/bsh/iot/dto/bshp/v1/ItemInfoV1Dto.java
package net.softbell.bsh.iot.dto.bshp.v1;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 * @Author : Bell(bell@softbell.net)
 * @Description : BSHPv1 전용 Items Info DTO
 */
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class ItemInfoV1Dto
{
	private Byte controlMode;
	private Byte itemIndex;
	private Integer itemCategory;
	private Integer itemType;
	private String itemName;
}

File: src/main/kotlin/net/softbell/bsh/iot/dto/bshp/v1/ItemInfoV1Dto.kt
package net.softbell.bsh.iot.dto.bshp.v1

/**
 * @Author : Bell(bell@softbell.net)
 * @Description : BSHPv1 전용 Items Info DTO
 */
data class ItemInfoV1Dto (
    var controlMode: Byte,
    var itemIndex: Byte,
    var itemMode: Int,
    var itemCategory: Int,
    var itemType: Int,
    var itemName: String
)